libs/postgres_ffi/src/lib.rs

```diff
@@ -28,6 +33,8 @@ macro_rules! postgres_ffi {
             pub mod waldecoder;
             pub mod xlog_utils;
 
+            pub const PG_MAJORVERSION: &str = stringify!($version);
```

Comment:
In PostgreSQL, `PG_MAJORVERSION` is just `14` or `15`, but here it's `v14` or `v15`. That seems confusing.

Reply:
Fair enough.
Will `PG_MODULE_VERSION` be better?
This is used mostly in postgres_ffi unit tests to determine correct bin and lib paths.